spacy/tests/lang/sa/test_text.py

```diff
@@ -0,0 +1,22 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import pytest
+
+
+def test_sa_tokenizer_handles_long_text(sa_tokenizer):
+    text = """नानाविधानि दिव्यानि नानावर्णाकृतीनि च।।"""
+    tokens = sa_tokenizer(text)
+    assert len(tokens) == 6
+
+
+@pytest.mark.parametrize(
+    "text,length",
+    [
+        ("श्री भगवानुवाच पश्य मे पार्थ रूपाणि शतशोऽथ सहस्रशः।", 9,),
+        ("गुणान् सर्वान् स्वभावो मूर्ध्नि वर्तते ।", 6),
+    ],
+)
+def test_sa_tokenizer_handles_cnts(sa_tokenizer, text, length):
+    tokens = sa_tokenizer(text)
+    assert len(tokens) == length
```

Comment:
Perhaps you could add a test here for the `like_num` functionality as well? See https://github.com/explosion/spaCy/blob/master/spacy/tests/lang/es/test_text.py#L56 for inspiration - you could keep most of it, but swap the Spanish words for relevant Sanskrit ones. Having a few `False` cases is always good, too.